src/common/OverlayManager.cpp

```diff
@@ -101,7 +102,7 @@ void OverlayManager::FireMouseEvent(const ultralight::MouseEvent& evt) {
 
       focused_overlay_ = hovered_overlay_;
 
-      if (window_focused_)
+      if (window_focused_ && focused_overlay_)
```

Comment:
I'm not sure if the `window_focused_` here was intended, or whether this is a "typo" and should've been a `focused_overlay_` check instead